public/assets/img/ecosystem/Dapp.tsx

```diff
@@ -1,32 +1,50 @@
-import Image from "next/image";
-import WalletAppsIcon from "./wallet-apps.png";
-import WalletAppsIconHover from "./wallet-apps-hover.png";
+import { Video } from "@components/commons/Video";
 
 interface WalletAppsProps {
   id: string;
-  className?: string;
 }
 
 export function WalletApps(props: WalletAppsProps): JSX.Element {
-  const { id, className } = props;
+  const { id } = props;
+
   return (
-    <Image
-      src={WalletAppsIcon}
-      alt="Wallet Apps Icon"
-      id={id}
-      className={className}
-    />
+    <div
+      style={{
+        position: "absolute",
+        top: "-175px",
+        right: "-65px",
+      }}
+    >
+      <Video
+        poster=""
```

Comment:
can we add a poster?

Reply:
Can but because the poster is the first thing that the card will load before the video, especially on hover, it will look distorted